letta/orm/passage.py

```diff
@@ -1,49 +1,112 @@
-from datetime import datetime
-from typing import TYPE_CHECKING, Optional
+from typing import TYPE_CHECKING
+from sqlalchemy import Column, JSON, Index
+from sqlalchemy.orm import Mapped, mapped_column, relationship, declared_attr
+from sqlalchemy.types import TypeDecorator, BINARY
 
-from sqlalchemy import JSON, Column, DateTime, ForeignKey, String
-from sqlalchemy.orm import Mapped, mapped_column, relationship
+import numpy as np
+import base64
 
-from letta.config import LettaConfig
-from letta.constants import MAX_EMBEDDING_DIM
-from letta.orm.custom_columns import CommonVector
 from letta.orm.mixins import FileMixin, OrganizationMixin
 from letta.orm.source import EmbeddingConfigColumn
 from letta.orm.sqlalchemy_base import SqlalchemyBase
+from letta.orm.mixins import AgentMixin, FileMixin, OrganizationMixin, SourceMixin
 from letta.schemas.passage import Passage as PydanticPassage
 from letta.settings import settings
 
+from letta.config import LettaConfig
+from letta.constants import MAX_EMBEDDING_DIM
+
 config = LettaConfig()
 
 if TYPE_CHECKING:
     from letta.orm.organization import Organization
+    from letta.orm.agent import Agent
+
+
+class CommonVector(TypeDecorator):
```

Comment:
i think this gets defined in `custom_columns.py`, let's moved shared definitions there